smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/endpoints/EndpointResolverGenerator.kt

```diff
@@ -9,111 +9,59 @@ import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.model.node.Node
 import software.amazon.smithy.rulesengine.language.EndpointRuleSet
 import software.amazon.smithy.rulesengine.traits.EndpointRuleSetTrait
-import software.amazon.smithy.swift.codegen.Dependency
-import software.amazon.smithy.swift.codegen.MiddlewareGenerator
-import software.amazon.smithy.swift.codegen.SwiftDependency
 import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.middleware.EndpointResolverMiddleware
 import software.amazon.smithy.swift.codegen.model.getTrait
 import software.amazon.smithy.swift.codegen.swiftmodules.ClientRuntimeTypes
 import software.amazon.smithy.swift.codegen.swiftmodules.SmithyHTTPAPITypes
-import software.amazon.smithy.swift.codegen.utils.toLowerCamelCase
 
 /**
  * Generates a per/service endpoint resolver (internal to the generated SDK)
  */
 class EndpointResolverGenerator(
     private val partitionDefinition: Symbol,
-    private val dependency: Dependency,
-    private val endpointResolverMiddleware: (
-        writer: SwiftWriter,
-        inputSymbol: Symbol,
-        outputSymbol: Symbol,
-        outputErrorSymbol: Symbol
-    ) -> EndpointResolverMiddleware
 ) {
     fun render(ctx: ProtocolGenerator.GenerationContext) {
         val rootNamespace = ctx.settings.moduleName
 
         val ruleSetNode = ctx.service.getTrait<EndpointRuleSetTrait>()?.ruleSet
         val ruleSet = if (ruleSetNode != null) EndpointRuleSet.fromNode(ruleSetNode) else null
 
-        ctx.delegator.useFileWriter("./$rootNamespace/EndpointResolver.swift") {
-            it.addImport(dependency.target)
-            val endpointParamsGenerator = EndpointParamsGenerator(ruleSet)
-            endpointParamsGenerator.render(it, ctx)
-        }
-
-        ctx.delegator.useFileWriter("./$rootNamespace/EndpointResolver.swift") {
+        ctx.delegator.useFileWriter("./$rootNamespace/Endpoints/EndpointResolver.swift") {
```

Comment:
This file now is written to the `Endpoints/` subdirectory inside the service client.